cross/acl/PLIST

```diff
@@ -0,0 +1,7 @@
+bin:bin/chacl
+bin:bin/getfacl
+bin:bin/setfacl
+lnk::libexec/libacl.so
+lnk:lib/libacl .so
```

Comment:
is the leading space before the extension really needed?

Reply:
The space for sure is not needed: it is better to remove it